src/meds_etl/flat.py

```diff
@@ -346,14 +463,16 @@ def convert_flat_to_meds(
     random.shuffle(parquet_tasks)
 
     if num_proc != 1:
-        with mp.get_context("spawn").Pool(num_proc, maxtasksperchild=1) as pool:
-            metadata_columns_set = set()
-            for columns in pool.imap_unordered(get_csv_columns, csv_tasks):
-                metadata_columns_set |= columns
-            for columns in pool.imap_unordered(get_parquet_columns, parquet_tasks):
-                metadata_columns_set |= columns
-
-            metadata_columns_set -= KNOWN_COLUMNS
+        with mp.get_context("spawn").Pool(num_proc) as pool:
+            metadata_columns_info = dict()
```

Comment:
This isn't quite optima. We need to double check for conflicting column types and throw an error. 

I'm fine keeping this for now though, as it's an improvement over the old code and we can resolve this better later.

Reply:
By conflicting column types, I mean people who have incorrectly had one type in some files but another type in others.

Reply:
It's a good point. I started something along these lines but decided to punt for now. Added a TODO so we remember to revisit.